core/modules/views/plugins/views_plugin_display.inc

```diff
@@ -2537,10 +2537,9 @@ class views_plugin_display extends views_plugin {
         if (!empty($this->view->exposed_raw_input)) {
           $url_options['query'] = $this->view->exposed_raw_input;
         }
-        $theme = views_theme_functions('views_more', $this->view, $this->display);
```

Comment:
I would keep this line and modify it like this:

`$theme = views_theme_functions('more_link', $this->view, $this->display);`

Reply:
This views_theme_functions() is what makes people love views and why it's so easy to extend it through templates.

Basically, when you call `$theme = views_theme_functions('more_link', 'view_custom', 'display_custom');`, the value of `$theme` will be: `more_link__view_custom__display_custom`.

Then, the templates that will be available are:
* `more-link--view-custom--display-custom.tpl.php`
* `more-link--view-custom.tpl.php`
* `more-link.tpl.php`